controllers/machinehealthcheck_targets.go

```diff
@@ -26,12 +26,30 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
+	ownerControllerKind      = "MachineSet"
+	nodeMasterLabel          = "node-role.kubernetes.io/master"
+	machineControlPlaneLabel = "cluster.x-k8s.io/control-plane"
+
+	// Event types
+
+	// EventSkippedMaster is emitted in case an unhealthy node (or a machine
+	// associated with the node) has Master role and reboot remediation strategy
```

Comment:
can we drop the "and reboot remediation strategy is not enabled" bit? it's meaningless here.

Reply:
Good spot, fixed